### DISCUSSION TITLE
using harbor to store ai models

### DISCUSSION THREAD

**Discussion by TheCodingSheikh** [goharbor/harbor#21358] [Q&A] [answered]
i watched a talk in Kubecon, where they used harbor to store hugging face artifacts, i'v looked everywhere and could find any resource, is this feature implemented or not yet ?

**Comment by chlins** [comment]
Please watch this issue: https://github.com/goharbor/harbor/issues/21229

**Reply by wy65701436** [reply, marked as answer]
The idea we're presenting to the community is a preliminary concept for managing AI models using OCI and Harbor. This is not currently part of the official roadmap but rather a demonstration of potential. As @chlins mentioned earlier, Harbor is collaborating with Ant Group to define a model specification, with the aim of pushing it toward becoming an industry standard.

**Reply by TheCodingSheikh** [reply]
It's clea now, thank you all, @wy65701436 @chlins 
Hopefully we see this in the near future
